crates/concolor/src/color/mod.rs

```diff
@@ -108,6 +108,11 @@ fn init() -> usize {
             flags |= InternalFlags::TRUECOLOR;
         }
     }
+    #[cfg(not(feature = "term"))]
+    {
+        // Default to assuming the terminal supports color (or the user doesn't care)
+        flags |= InternalFlags::TERM_SUPPORT;
+    }
```

Comment:
note: so long as we are not exposing the more specific details, this should be fine.  If we ever expose more details, then I would rather we report to the user that we don't know.